src/commands/templates/add.ts

```diff
@@ -0,0 +1,67 @@
+import * as fs from 'fs';
+import { runCommand, readJsonInput } from '../../util/command';
+import { detailOutputFormat } from './util';
+
+export = {
+	command: 'add',
+	desc: 'Add template',
+	builder: yargs => {
+		yargs.options({
+			'kind': {
+				desc: 'The job template kind',
+				type: 'string',
+				default: 'transfer',
+				demand: true
+			},
+
+			'name': {
+				desc: 'The job template name',
+				type: 'string'
+			},
+
+			'auto': {
+				desc: 'Schedule the job as automatic',
+				type: 'boolean'
+			},
+
+			'manual': {
+				desc: 'Schedule the job as manual',
+				type: 'boolean'
+			},
+
+			'options-input': {
+				alias: 'in',
+				desc: 'Read JSON options from stdin',
+				type: 'boolean'
+			},
+
+			'options-file': {
+				alias: 'file',
+				desc: 'The options JSON file',
+				type: 'string'
+			}
+		})
+	},
+	handler: argv => {
+		runCommand(argv, async (client, output) => {
+			let template: any = {
+				kind: argv.kind,
```

Comment:
use spread syntax to pass along argv properties:  `...argv`

Reply:
That will include a ton of additional parameters that are not related to the job, which we probably don't want. Especially since that would include authentication details.